api/maven-api-model/pom.xml

```diff
@@ -63,26 +63,26 @@ under the License.
         </executions>
       </plugin>
       <plugin>
-        <groupId>org.apache.maven</groupId>
-        <artifactId>modello-plugin-velocity</artifactId>
-        <configuration>
-          <version>4.2.0</version>
-          <models>
-            <model>src/main/mdo/maven.mdo</model>
-          </models>
-          <templates>
-            <template>src/main/mdo/model.vm</template>
-          </templates>
-          <params>
-            <param>packageModelV4=org.apache.maven.api.model</param>
-          </params>
-        </configuration>
+        <groupId>org.codehaus.modello</groupId>
+        <artifactId>modello-maven-plugin</artifactId>
         <executions>
           <execution>
-            <id>modello</id>
+            <id>velocity</id>
             <goals>
               <goal>velocity</goal>
             </goals>
+            <configuration>
+              <version>4.2.0</version>
+              <models>
+                <model>src/main/mdo/maven.mdo</model>
+              </models>
+              <templates>
+                <template>src/main/mdo/model.vm</template>
+              </templates>
```

Comment:
Stupid question, do we need this separation between main and test for models at all?

Reply:
What do you mean exactly ? I don't understand.

Reply:
What is the purpose of having those models in `main` if we will never distinct between `main` and `test`?

Reply:
1. it has been our convention for ever https://github.com/apache/maven/tree/maven-3.9.x/maven-model/src/main/mdo
2. it's really main code that it generated, then storing in `src/main` makes sense

Reply:
Agreed.